pysph/examples/flow_past_cylinder_2d.py

```diff
@@ -0,0 +1,323 @@
+        from pysph.base.kernels import QuinticSpline
+        from pysph.sph.wc.transport_velocity import (
+            SetWallVelocity, MomentumEquationPressureGradient,
+            SolidWallNoSlipBC, VolumeSummation,
+            MomentumEquationViscosity
+        )
+        data = load(self.output_files[0])
+        solid = data['arrays']['solid']
+        fluid = data['arrays']['fluid']
+
+        prop = ['awhat', 'auhat', 'avhat', 'wg', 'vg', 'ug', 'V', 'uf', 'vf',
+                'wf', 'wij', 'vmag']
+        for p in prop:
+            solid.add_property(p)
+            fluid.add_property(p)
+        # We find the force of the solid on the fluid and the opposite of that
+        # is the force on the solid. Note that the assumption is that the solid
+        # is far from the inlet and outlet so those are ignored.
+        equations = [
+            Group(
+                equations=[
+                    SetWallVelocity(dest='fluid', sources=['solid']),
+                    VolumeSummation(dest='fluid', sources=['fluid', 'solid']),
+                    VolumeSummation(dest='solid', sources=['fluid', 'solid']),
+                ], real=False),
+            Group(
+                equations=[
+                    # Pressure gradient terms
+                    MomentumEquationPressureGradient(
+                        dest='solid', sources=['fluid', 'solid'], pb=p0),
+                    MomentumEquationViscosity(
+                        dest='solid', sources=['fluid', 'solid'], nu=self.nu),
+                    SolidWallNoSlipBC(
+                        dest='solid', sources=['fluid'], nu=self.nu),
+                    ], real=True),
+        ]
+        sph_eval = SPHEvaluator(
+            arrays=[solid, fluid], equations=equations, dim=2,
+            kernel=QuinticSpline(dim=2)
+        )
+        t, cd, cl = [], [], []
+        for sd, arrays in iter_output(self.output_files):
+            fluid = arrays['fluid']
+            solid = arrays['solid']
+            for p in prop:
+                solid.add_property(p)
+                fluid.add_property(p)
+            t.append(sd['t'])
+            sph_eval.update_particle_arrays([solid, fluid])
+            sph_eval.evaluate()
+            cd.append(np.sum(solid.au*solid.m)/(0.5*rho*umax**2*diameter))
+            cl.append(np.sum(solid.av*solid.m)/(0.5*rho*umax**2*diameter))
+        t, cd, cl = list(map(np.asarray, (t, cd, cl)))
+        # Now plot the results.
+        import matplotlib
+        matplotlib.use('Agg')
+        from matplotlib import pyplot as plt
+        plt.figure()
+        plt.plot(t, cd, label=r'$C_d$')
+        plt.plot(t, cl, label=r'$C_l$')
+        plt.xlabel(r'$t$')
+        plt.ylabel('Force')
+        plt.legend()
+        fig = os.path.join(self.output_dir, "force_vs_t.png")
+        plt.savefig(fig, dpi=300)
+        plt.close()
+        return t, cd, cl
+
+    def customize_output(self):
+        self._mayavi_config('''
```

Comment:
Is this needed?  Or do you think it serves as a useful example?

Reply:
No not a useful example. removed